spec/models/immunisation_import_row_spec.rb

```diff
@@ -782,6 +782,51 @@
 
       it { should be_valid }
     end
+
+    context "validating clinic name" do
+      let(:valid_clinic_data) do
+        valid_data.merge(
+          "CARE_SETTING" => "2",
+          "DATE_OF_VACCINATION" => session.dates.first.strftime("%Y%m%d"),
+          "SESSION_ID" => session.id.to_s,
+          "ORGANISATION_CODE" => organisation.ods_code,
+          "PERFORMING_PROFESSIONAL_EMAIL" => create(:user).email
+        )
+      end
+
+      let(:session) { create(:session, organisation:, programmes:) }
+
+      before { create(:community_clinic, name: "A clinic", organisation:) }
+
+      context "with an existing community clinic" do
+        let(:data) { valid_clinic_data.merge("CLINIC_NAME" => "A clinic") }
+
+        it "is matching" do
+          expect(immunisation_import_row).to be_valid
+        end
```

Comment:
```suggestion
        it { should be_valid }
```

Because the `subject` is set to `immunisation_import_row`, we can simplify this code even further. However, it may not always be appropriate if we want to call out something specific in the naming of the `it`, so I don't mind either way.

Reply:
Removed the subject and still have it more verbose to be consistent with other 'validation' unit tests in the block